.gitignore

```diff
@@ -0,0 +1,2 @@
+.idea/
+.idea/workspace.xml
```

Comment:
If `.idea` is excluded then you don't need to second entry.

Reply:
done

Reply:
Are you sure? I still see the second entry.